MapImageEngine/source/resources/strings/deu.ini

```diff
@@ -0,0 +1,49 @@
+plugin-loader.loading="MapImageEngine Aktivierung..."
+plugin-loader.reloading="MapImageEngine neuladen..."
+plugin-loader.info-instruction="Notiz #1: Anleitung im Plugin Ordner"
+plugin-loader.info-long-loading="Notiz #2: Bei neuen Bilder oder bei einem Update kann es sein dass, das Laden länger dauert!"
+plugin-loader.info-1.1-update="Notice #3: nach der Installation von MapImageEngine v1.1 musst du alle Bilder neusetzen"
+plugin-loader.loaded="MapImageEngine Aktiviert!"
+plugin-loader.reloaded="MapImageEngine Neugeladen!"
+
+image-loader.prefix="Bild \"%s\": "
+image-loader.succesfull="Bild geladen"
+image-loader.err-name-exists="Dieser Name ist bereits vorhanden"
+image-loader.err-image-exists="Bild Bereits vorhanden"
+image-loader.err-corrupted="Datei beschädigt"
+image-loader.err-unsupported-api="API inkompatibel. Bitte Update das Plugin. (und konvertiere die Bilder erneut falls es nicht geholfen hat)"
+image-loader.converted="Bild wurde in einem neuen Format konvertiert! Originale Bild in  'images/old_images' Ordner"
+image-loader.not-converted="Bild kann nicht Konvertiert werden! Bitte Konvertiere das Original Bild nochmal auf drr Webseite oder mit dem MIE_Converter Plugin"
+
+command.desc="MapImageEngine Befehle"
+command.desc.list="Liste der Verfügbaren Bilder"
+command.desc.place="Betrete Platziermodus"
+command.desc.exit="Verlasse Platziermodus"
+
+command.usage="Nutze/Nutzung: "
+
+command.in-game="Führe den Befehl im Spiel aus"
+
+command.list="Verfügbare Bilder: "
+command.list.no-images="Du hast noch keine Bilder"
+
+command.place.usage="<Bildname> [Flagge]"
+command.place.usage.flags="Verfügbare Flaggen: "
+command.place.usage.flags.pretty="Quadrate für bequeme Orientierung anzeigen"
+command.place.usage.flags.auto="Automatischer Modus (Platziert ein Bild in zwei Klicks)"
+command.place.not-found="Bild \"%s\" wurde nicht gefunden!"
+command.place.placing="Platziere Bild!"
+command.place.click-top-left="Klicke auf die obere linke Ecke..."
+command.place.click-bottom-right="Klicke auf die untete rechte Ecke..."
+command.place.placing-info="Die Platzierung beginnt von der oberen linken Ecke (x * y)"
+command.place.click="Klicke auf den Rahmen (%d * %d)"
+command.place.not-frame="Das ist kein Rahmen!"
+command.place.width-not-match="Breite entspricht nicht der Breite des Bildes!"
+command.place.height-not-match="Höhe entspricht nicht der Höhe des Bildes!"
+command.place.invalid-upper-corner="Die obere Ecke kann nicht niedriger sein als die untere Ecke!"
+command.place.no-frames="Du musst die Rahmen in den ausgewählten Bereich stellen! Lege alle Rahmen und versuchenes erneut
```

Comment:
Trailing quotation marks are missing here.

Reply:
I will rework it when it is possible

Reply:
I will rework it when it is possible

Reply:
So, it's not ready yet?

Reply:
Yea "Flagge" is a Bad Translation can i or you change it to "Modus" this is better

Reply:
Sorry but I think it's far away from ready, there are still many grammar and spelling mistakes. Of course we could also correct them after merging, although it would be better before.

Reply:
I prefer that it is ready before merging, so I'll wait